clang/test/CXX/drs/dr28xx.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clang_cc1 -std=c++20 -verify=expected %s
```

Comment:
Can you copy RUN lines from `dr26xx.cpp`, adding verify prefixed you need in your test (`since-cxx20`)?